includes/class-wc-install.php

```diff
@@ -605,7 +605,7 @@ private static function get_schema() {
 ) $collate;
 CREATE TABLE {$wpdb->prefix}woocommerce_log (
   log_id BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
-  timestamp datetime NOT NULL,
+  timestamp datetime(6) NOT NULL DEFAULT NOW(5),
```

Comment:
This is going to be problematic as it appears to be _invalid syntax_ in MySQL `<5.6`.

You can see it failing in the automated tests:

> WordPress database error You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '(6) NOT NULL DEFAULT NOW(5),

https://travis-ci.org/woocommerce/woocommerce/jobs/230864195#L316

Maybe a [version comment](https://dev.mysql.com/doc/refman/5.5/en/comments.html) will work here, something like `timestamp datetime/*!50600 (6) */ NOT NULL DEFAULT NOW/*!50600 (5) */` (untested).